Userland/Libraries/LibVideo/CodecID.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2023, Stephan Vedder <stephan.vedder@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/HashFunctions.h>
+#include <AK/Time.h>
+#include <AK/Traits.h>
+#include <AK/Types.h>
+#include <AK/Variant.h>
+
+namespace Video {
+
+enum class CodecID : u32 {
+    Unknown,
+    // On2 / Google
+    VP8,
+    VP9,
+    // MPEG
```

Comment:
This comment is technically incorrect, maybe make it two comments for `// ITU-T` and `// MPEG`